test/controllers/security.test.js

```diff
@@ -1067,8 +1065,7 @@ describe('Security Controller', () => {
               controller: 'security',
               action: 'searchRoles',
               body: {controllers: ['foo', 'bar']},
-              from: undefined,
-              size: undefined
+              opt: 'in',
```

Comment:
I personally prefer to add a test than to modify an existing one even if you copy/paste it